pom.xml

```diff
@@ -50,7 +50,7 @@
         <dep.airlift.version>0.148</dep.airlift.version>
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.slice.version>0.29</dep.slice.version>
-        <dep.aws-sdk.version>1.11.30</dep.aws-sdk.version>
+        <dep.aws-sdk.version>1.11.165</dep.aws-sdk.version>
```

Comment:
@electrum I can set up hive with metastore, and run ```TestHiveClientS3```. What do you mean by direct integration test with S3? Running ```PrestoS3FileSystem``` without hive and metastore? Would be interested to tackle this challenge.